src/styles.css

```diff
@@ -2,6 +2,14 @@
 @tailwind components;
 @tailwind utilities;
 
+input.ng-invalid.ng-touched {
+  @apply tce-border-red-600 dark:tce-border-red-700 tce-border-2 tce-m-[-1px]
```

Comment:
The darker border is a bit harder to see on dark mode but matches the background for the error message. I think that the border is a bit difficult to see either way on dark mode and the error message would be the main indication of an error so hopefully that's ok as long as the error message itself is clear.

Reply:
My only thought is we might want to put the `dark:` parts into the `package-styles.css file` instead. Only because the SL site does not support dark mode, and you'll then get the red background if you have it set in your preferences:

![image](https://github.com/user-attachments/assets/495f58fc-88f6-484a-b34e-939ddf08b24c)

Reply:
Done.